### PR TITLE
de: security guide

### PR DESCRIPTION
**Pull request by kradan**
- [X] pages/security/message-security/openpgp/gpg-keys/de.text
- [X] pages/security/device-security/device-encryption/de.md
- [X] pages/security/device-security/software-safety/de.md
- [X] pages/security/device-security/mobile-devices/de.md
- [X] review and proofreading

### CODE REVIEW

**Comment by micah** [comment]
Hi @kradan -- Is this is still a WIP?

**Comment by kradan** [comment]
> Hi @kradan -- Is this is still a WIP?

it actually is, there are untranslated parts, but i don' know when i
can get to it.

- pages/security/message-security/openpgp/gpg-keys/de.text [Gpg4win]
- pages/security/device-security/device-encryption/de.md
- pages/security/device-security/software-safety/de.md
- pages/security/device-security/mobile-devices

**Comment by dtto** [comment]
Hi @kradan I'm trying to catch up on some of these. Is this in a position to merge? Or still a WIP?

**Comment by kradan** [comment]
i'm sorry for the late reply. so far nothing changed here. if interested, let us know here when you start translating a file.

**Comment by kradan** [comment]
@elijh @micah content question for *device-encryption*:
> With device encryption the contents of your device's storage--the part that contains the operating system, programs you have installed, and your personal data--are scrambled so that it cannot be accessed when the device is off or when you are logged out.

I find it misleading to believe that my data is not accessible when i am logged out (of my desktop session), because keys are still in RAM and could be read by privileged applications and hence tend to leave out the "or when you are logged out" in the translation - if that's ok?

**Comment by kradan** [comment]
I'm done and would be happy about some proofreading.
@RainerBielefeld, do you want to have a look?

File: pages/security/human-security/message-hygiene/de.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Der Hauptgrund, dass Emails die Hauptangriffsmöglichkeit einer Organisation sin
 
 Zur Wiederholung: Die "Von"-Adresse kann von allen beliebig manipuliert werden, sodass eine Email so aussieht, als käme sie von einer anderen Person.
```
Comment (RainerBielefeld) [inline pages/security/human-security/message-hygiene/de.md]: sodass -->
so dass

File: pages/security/human-security/message-hygiene/de.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Der Hauptgrund, dass Emails die Hauptangriffsmöglichkeit einer Organisation sin
...
 
-Weil die Herkunft einer Nachricht so schwer zu verifizieren ist, ist der Posteingang eine übliche Quelle für **[Phishing-Angriffe -> phishing]** und **[Schadsoftware->malware]**.
+Weil die Herkunft einer Nachricht so schwer zu verifizieren ist, ist der Posteingang eine übliche Quelle für **[Phishing-Angriffe -> phishing]** und **[Schadsoftware->viruses]**.
 
 * **Phishing** ist when someone sends an email claiming to be an entity they are not, and uses this deception to get information. They can be after social security numbers, bank information, passwords, or other sensitive information.
```
Comment (RainerBielefeld) [inline pages/security/human-security/message-hygiene/de.md]: Translation:
Von **Phishing** spricht man, wenn in einer Email eine falsche Absender-Identität vorgegaukelt und diese Täuschung benutzt wird, um an persönliche Informationen des Empfängers zu gelangen. Das können Kontonummern, Zugangsdaten und Passwörter, Sozialversicherungsnummern und viele andere vertrauliche Informationen sein.
End of translation

BTW:" I do not understand **[Phishing-Angriffe -> phishing]** und **[Schadsoftware->malware]**." Those arehyperlinks which will work in the final Help environment?

File: pages/security/human-security/message-hygiene/de.md (hunk 1 above)
Comment (RainerBielefeld) [reply pages/security/human-security/message-hygiene/de.md]: Some more comments concerning the compete file with suggestions for corrections:
Quele   -->
Quelle

Unerwarte   -->
Unerwartete

better   -->
bessere

I hope this helps,Ican help with some more translation next week.

Comment (kradan) [reply pages/security/human-security/message-hygiene/de.md]: thanks!

File: pages/security/human-security/message-hygiene/de.md (hunk 2 above)
Comment (kradan) [reply pages/security/human-security/message-hygiene/de.md]: yes, they point to [/de/email/scams/phishing](https://riseup.net/de/email/scams/phishing) and [/de/email/scams/viruses](https://riseup.net/de/email/scams/viruses)

**Comment by disturbio** [comment]
Hi,

Would it be possible to submit this PR against the no-masters branch?

**Comment by micah** [comment]
> @elijh @micah content question for _device-encryption_:
> 
> > With device encryption the contents of your device's storage--the part that contains the operating system, programs you have installed, and your personal data--are scrambled so that it cannot be accessed when the device is off or when you are logged out.
> 
> I find it misleading to believe that my data is not accessible when i am logged out (of my desktop session), because keys are still in RAM and could be read by privileged applications and hence tend to leave out the "or when you are logged out" in the translation - if that's ok?

I agree with you, that seems like a better way as you suggest!

**Comment by disturbio** [comment]
This was merged, except for pages/security/device-security/device-encryption/de.md which was alredy translated.

**Pull request closed without merging**